samples/nrf9160/gps/src/main.c

```diff
@@ -37,12 +37,12 @@ nrf_gnss_data_frame_t last_fix;
 
 void bsd_recoverable_error_handler(uint32_t error)
 {
-	printf("Err: %lu\n", error);
+	printf("Err: %u\n", error);
```

Comment:
This actually depends on what standard library is used. Zephyrs minimal libc indeed defines `uint32_t` as `unsigned int`, yet the newlib defines it as `unsigned long`, and the sample appears to use newlib. Perhaps we should use `"PRIu32"` to be on a safe side?

Reply:
> Perhaps we should use "PRIu32" to be on a safe side?

I would rather cast to unsigned long in the printf statement if that is the case -- OK for you?

Reply:
Sure